src/SampledCorrelations/CorrelationSampling.jl

```diff
@@ -1,22 +1,19 @@
-function observable_values!(buf, sys::System{N}, ops; apply_g = true) where N
+function observable_values!(buf, sys::System{N}, ops) where N
     if N == 0
-        for site in eachsite(sys), (i, op) in enumerate(ops)
-            dipole = sys.dipoles[site]
-            if apply_g
-              dipole = sys.gs[site] * dipole
+        for (i, op) in enumerate(ops)
+            for site in eachsite(sys)
+                A = observable_at_site(op,site)
+                dipole = sys.dipoles[site]
+                buf[i,site] = A * dipole
```

Comment:
Just an FYI: More generally, we will eventually want _nonlinear_ maps of dipole for observables. This could naturally be achieved with Stevens polynomials. But that discussion can happen later.

Reply:
Ah yes I see. Observables are simply different for dipole mode vs SU(N) it seems. (Mainly because dipole mode e.g. doesn't include quadrupoles when S=1)